tests/dummy/app/templates/priority.hbs

```diff
@@ -1,5 +1,5 @@
 {{format-markdown "
-## Priority
+## Priority (6.x and below)
```

Comment:
Keep this page with documentation and interactive widgets/examples for 6.x

Basically deprecated/removed functionality for old propagation model/priority based responders moved from add-on code to dummy app for the purpose of interactive docs only.